### ISSUE TITLE
add a function gg_barbs to plot wind barbs

### ISSUE THREAD

**Issue by PanfengZhang** [eliocamp/metR#190]
### Tips for a helpful feature request:

In situations when draw wind vectors (or wind barbs) with python <https://matplotlib.org/stable/plot_types/arrays/barbs.html#sphx-glr-plot-types-arrays-barbs-py>, I would like to be able to draw wind vectors (or wind barbs) using ggplot2 and metR package. Can you add a new function **gg_barbs** in the metR package to implement this feature?

Thank you very, very much.

The wind vector illustration, which is drawed by matplotlib of python, is as follows:
![Figure_1](https://github.com/eliocamp/metR/assets/20354811/284a538f-c64b-453c-a7f6-865c93cadc7b)

**Comment by eliocamp** [comment]
Yes, adding barbs has been on the docket for some time (#162) but I haven't had the time yet to implement it.
